damus/Util/Translator.swift

```diff
@@ -68,10 +70,9 @@ public struct Translator {
 
         struct RequestBody: Encodable {
             let text: [String]
-            let source_lang: String
             let target_lang: String
         }
-        let body = RequestBody(text: [text], source_lang: sourceLanguage.uppercased(), target_lang: targetLanguage.uppercased())
```

Comment:
This is a notable change in behaviour. Previously, we were relying on Apple's predicted language as the source language. We're now just letting DeepL do it on their side as it's integrated within the translate API, and I'm hoping that DeepL's online prediction is better than Apple's offline one.